entgql/internal/todo/gen_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019-present Facebook
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package todo_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent/entc"
+	"entgo.io/ent/entc/gen"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGeneratedSchema(t *testing.T) {
```

Comment:
Nice, but don't we already test this here: https://github.com/ent/contrib/blob/master/entgql/schema_test.go ?
I don't mind keeping it here.

cc @giautm

Reply:
schema_test runs tests by calling `buildTypes`. I added logic to `schemaHook`, which executes before `buildTypes`